src/main/kotlin/notification/domain/Notification.kt

```diff
@@ -2,4 +2,4 @@ package notification.domain
 
 import shared.domain.Identifier
 
-class Notification(val id: Identifier)
+class Notification(val noteId: Identifier)
```

Comment:
I suppose that you didn't add the Date member because you're doing TDD and, in this use case, isn't needed. If it's the case well done! 😺